### PR TITLE
instructions for upgrades to v0.20.6

### PR DESCRIPTION
**Pull request by mattkasun**

### CODE REVIEW

File: upgrades.rst [hunk 1]
```diff
@@ -5,7 +5,23 @@ Upgrades
 Introduction
 ===============
 
-As of 0.13.0, upgrading Netmaker is a manual process. This is expected to be automated in the future, but for now, is still a relatively straightforward process. 
+As of 0.13.0, and up 0.17.1 upgrading Netmaker is a manual process. As of 0.20.6 the upgrade process from 0.17.1 to current vesrsion has been automated.
```
Comment (theguy951357) [inline upgrades.rst:8]: typo vesrsion -> version

File: upgrades.rst [hunk 2]
```diff
@@ -5,7 +5,23 @@ Upgrades
...
 ===============
 
-As of 0.13.0, upgrading Netmaker is a manual process. This is expected to be automated in the future, but for now, is still a relatively straightforward process. 
+As of 0.13.0, and up 0.17.1 upgrading Netmaker is a manual process. As of 0.20.6 the upgrade process from 0.17.1 to current vesrsion has been automated. 
+
+Notes:
+================================
+1. The server must be upgraded before any clients.  
+2. If upgrading to CE version, any relays will be lost
+3. If upgrading from EE to EE, a new licence key and tennet id must be obtained from https://app.netmaker.io
```
Comment (theguy951357) [inline upgrades.rst:14]: tennet -> tenant

**Review by theguy951357 — CHANGES_REQUESTED** [review]

**Review by theguy951357 — APPROVED** [review]

File: upgrades.rst [hunk 3]
```diff
@@ -5,7 +5,23 @@ Upgrades
...
+2. If upgrading to CE version, any relays will be lost
+3. If upgrading from EE to EE, a new licence key and tenant id must be obtained from https://app.netmaker.io
+4. Relayed clients must be updated before the relays
+5. As the netmaker netclient node is updated when updating the server, if the netmaker node was a relay, the relayed nodes will have to be reset
+6. As each netclient is updated, a new host, nodes, and gateways will be visible in the netmaker UI.
+
+
+Steps:
+================================
+1. Download the nm-upgrade.sh script from https://fileserver.netmaker.io/upgrades/nm-upgrade.sh; make the script executable and run it
```
Comment (abhishek9686) [inline upgrades.rst:22]: can you mention to run the script on the server

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

**Review by Aceix — APPROVED** [review]

File: upgrades.rst
Comment (abhishek9686) [inline upgrades.rst:1]: we should also mention about extclients need to be re-created

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

**Review by theguy951357 — APPROVED** [review]

File: upgrades.rst [hunk 4]
```diff
@@ -5,7 +5,31 @@ Upgrades
...
+1. The server should be upgraded before any clients.  
+2. Relays will need to be recreated after the server and all clients are upgraded
+3. If upgrading from EE to EE, a new licence key and tennet id must be obtained from https://app.netmaker.io
+4. As each netclient is updated, a new host, nodes, and gateways (if applicable) will be visible in the netmaker UI.
+5. Extclient config files may have to be regenerated after the upgrade.
+
+
+Steps:
+================================
+1. Download the nm-upgrade.sh script from https://fileserver.netmaker.io/upgrades/nm-upgrade.sh; make the script executable and run it
```
Comment (abhishek9686) [inline upgrades.rst:21]: fix link `https://fileserver.netmaker.io/upgrade/nm-upgrade.sh`

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

**Review by abhishek9686 — APPROVED** [review]

**Pull request merged**